vocabularies/Common.json

```diff
@@ -1461,7 +1461,8 @@
             "$Kind": "ComplexType",
             "Property": {
                 "$Type": "Edm.PropertyPath",
-                "@Core.Description": "Sort property"
+                "@Core.Description": "Sort property",
+                "@Core.LongDescription": "May also resolve to an annotation with the term [`Aggregation.CustomAggregate`](https://oasis-tcs.github.io/odata-vocabularies/vocabularies/Org.OData.Aggregation.V1.html)\n            or [`Analytics.AggregatedProperty`](Analytics.md#AggregatedPropertyType). In this case the sort property is the dynamic property\n            contributed by the aggregation that is defined by the annotation."
```

Comment:
Sorry, this is a breaking change, you cannot do this.

Currently you can use these property paths to construct an `$orderby` query option, e.g. `$orderby=Size`.

This won't work with an annotation: `$orderby=@Aggregation.CustomAggregate#sum` is not what you want to achieve.